definitions/procedures/backup/offline/candlepin_db.rb

```diff
@@ -3,6 +3,7 @@ class CandlepinDB < ForemanMaintain::Procedure
     metadata do
       description 'Backup Candlepin DB offline'
       tags :backup
+      label 'backup-offline-candlepin-db'
```

Comment:
`:symbols` are being used for label and so far, there were in `:snake_case` form. The label is implicitly generated based on the file name and structure, so that it should not be needed.

Reply:
I've fixed the format. The reason I'm adding it is the procedures are called `CandlepinDB` which end up in labebel `candlepin-d-b`.